src/main/java/pl/nogacz/chess/application/menu/Undo.java

```diff
@@ -0,0 +1,94 @@
+package pl.nogacz.chess.application.menu;
+
+import java.util.ArrayList;
+import pl.nogacz.chess.application.ChessNotation;
+import pl.nogacz.chess.board.Board;
+import pl.nogacz.chess.board.Coordinates;
+import pl.nogacz.chess.pawns.PawnClass;
+
+public class Undo {
+
+    public Undo() {
+        String lastMove=ChessNotation.removeMovement();
+
+        if(lastMove.equals("")) {
+            System.out.println("No moves detected!");
+        }
+        else {
+            String computer = lastMove.substring(lastMove.indexOf(" ") + 1);
+            String player = lastMove.substring(0,lastMove.indexOf(" "));
+
+            removeMovement(computer);
+            removeMovement(player);
+        }
+    }
+
+    public static Coordinates getCoordinate(String s){
+        char letter = s.charAt(0);
+        int y = Integer.parseInt(s.substring(1));
+        int x = -1;
+        y--;
+        if(letter == 'a'){
```

Comment:
Bu harfler için bir enum tanımla. Bu metodun argument typeını da o enum yap.

Reply:
Hocam burada a-h enumları tanımlayıp karakterin hangisi olduğuna göre o enum cinsinden oluşturmak bana mantıklı gelmiyor.